server/utils/db/db_utils.py

```
@@ -3059,6 +3059,34 @@ def initialize_tables():
+                    END $$;
+                """)
+                cursor.execute("GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO aurora_app;")
+                cursor.execute("GRANT USAGE, SELECT ON ALL SEQUENCES IN SCHEMA public TO aurora_app;")
+                cursor.execute("GRANT USAGE ON SCHEMA public TO aurora_app;")
+                cursor.execute("ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT, INSERT, UPDATE, DELETE ON TABLES TO aurora_app;")
+                cursor.execute("ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT USAGE, SELECT ON SEQUENCES TO aurora_app;")
+                cursor.execute("GRANT aurora_app TO CURRENT_USER;")
+                conn.commit()
+                logging.info("Ensured aurora_app role exists with NOSUPERUSER NOBYPASSRLS.")
+            except Exception as e:
+                logging.error(f"FATAL: Error creating aurora_app role: {e}")
```

Ruff (0.15.20)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)